CHANGELOG.md

```diff
@@ -1,11 +1,23 @@
 
+<a name="v0.8.1"></a>
+## [v0.8.1](https://github.com/talos-systems/talos/compare/v0.8.0...v0.8.1) (2020-12-29)
+
+### Fix
+
+* networkd updates for Packet, hostname detection, console kernel arg
```

Comment:
Didn't we have 2 changes we wanted in this release?

Reply:
I even picked up three and had to add fourth one (to fight with the go-procfs changes), but as they're squashed... not sure what to do? three PRs? fix up change log manually?

Reply:
https://github.com/talos-systems/talos/commit/6eb8bf4d7a9a76238a6d181299e09f21c5612159

Reply:
ah, ok... no this should be fine.

Reply:
I usually have more than one commit in a back port, FWIW.

Reply:
yep, makes sense, I got stuck with 1 commit mindset, I'll do it better next time